### PR TITLE
feat: add Help/Support e-mail for instance/org

### LINKED ISSUE #2526: Help/Support E-MAil on Organisation
As a admin I am able to set a help / support email address on the instance and the organisations, which is shown to the end users in the login.

**Acceptance Criteria**
- [ ] I am able to set a help / support email on the instance
- [ ] I am able to set a help / support email on the organisation
- [ ] If a help / support email is set on the instance, I can see it in the login flow
- [ ] If a help / support email is set on my organisation, I can see it in the login flow

### PR DESCRIPTION
**Pull request by doncicuto**
As an admin I am now able to set a help / support email address on the instance and the organisations, which is shown to the end users in the login.

Acceptance Criteria

- [X] I am able to set a help / support email on the instance
- [X] I am able to set a help / support email on the organisation
- [X] If a help / support email is set on the instance, I can see it in the login flow
- [X] If a help / support email is set on my organisation, I can see it in the login flow

Here are some screenshots
instance_support_email

welcome_with_support_email

org_support_email

login_interface_text_footer

welcome_with_different_footer_email

This PR also fixes an issue where privacy policy links only could be assigned to instance, organizations where using instance defaults and those could not be overriden.

Closes #2526

```[tasklist]
### Definition of Ready
- [X] I am happy with the code
- [X] Short description of the feature/issue is added in the pr description
- [X] PR is linked to the corresponding user story
- [X] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [X] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [X] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [X] Functionality of the acceptance criteria is checked manually on the dev system.
```

### CODE REVIEW

**Comment by hifabienne** [comment]
@zitadel/engineers can one of you have a look at this?

**Comment by hifabienne** [comment]
Thanks @doncicuto we really appreciate all of your work

**Review by eliobischof — COMMENTED** [review]
LGTM :tada:  thank you a lot @doncicuto :pray: 
@peintnermax see my review comments. Thanks :slightly_smiling_face:

File: console/src/app/modules/policies/privacy-policy/privacy-policy.component.html [hunk 1]
```diff
@@ -36,6 +36,12 @@ <h2>{{ 'POLICY.PRIVACY_POLICY.TITLE' | translate }}</h2>
       <input cnslInput name="helpLink" formControlName="helpLink" />
       <template [ngTemplateOutlet]="templateRef" [ngTemplateOutletContext]="{ key: 'helpLink' }"></template>
     </cnsl-form-field>
+
+    <cnsl-form-field class="privacy-policy-formfield">
+      <cnsl-label>{{ 'POLICY.PRIVACY_POLICY.SUPPORTEMAIL' | translate }}</cnsl-label>
+      <input cnslInput name="supportEmail" formControlName="supportEmail" />
+      <template [ngTemplateOutlet]="templateRef" [ngTemplateOutletContext]="{ key: 'supportEmail' }"></template>
+    </cnsl-form-field>
```
[the remark this answers is not in the document]
Comment (peintnermax) [reply console/src/app/modules/policies/privacy-policy/privacy-policy.component.html:44]: yes. I'd use two grid columns such that we get a 2x2 layout. Can you change that please @doncicuto? Thx

File: console/src/app/modules/settings-list/settings-list.component.html [hunk 2]
```diff
@@ -53,6 +53,6 @@
     <cnsl-login-texts [serviceType]="serviceType"></cnsl-login-texts>
   </ng-container>
   <ng-container *ngIf="currentSetting === 'privacypolicy'">
-    <cnsl-privacy-policy [serviceType]="PolicyComponentServiceType.ADMIN"></cnsl-privacy-policy>
+    <cnsl-privacy-policy [serviceType]="serviceType"></cnsl-privacy-policy>
```
[the remark this answers is not in the document]
Comment (peintnermax) [reply console/src/app/modules/settings-list/settings-list.component.html:56]: yes.. we have org and instance settings.. this is a bug fix actually. Thx @doncicuto 👍

**Review by peintnermax — CHANGES_REQUESTED** [review]
LGTM.. I'd just change the grid layout to two columns.. thx for the nice work 👍

**Comment by doncicuto** [comment]
Thanks @peintnermax I'll change the grid layout as requested and update the PR once its finished

**Comment by doncicuto** [comment]
Hi @peintnermax, this is the new 2x2 grid  layout. Updating PR @adlerhurst 

image

**Review by peintnermax — APPROVED** [review]
would not have made 75% width but it's fine 👍 thx

**Comment by eliobischof** [comment]
Review and adjustments took ~3h

**Pull request merged**